js/libs/keycloak-admin-client/pom.xml

```diff
@@ -13,9 +13,36 @@
 
     <name>Keycloak JavaScript Admin Client</name>
     <description>A JavaScript client to interact with Keycloak's Administration API.</description>
+    <packaging>pom</packaging>
 
     <build>
         <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-deploy-plugin</artifactId>
+                <configuration>
+                    <skip>true</skip>
+                </configuration>
+            </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-antrun-plugin</artifactId>
+                <version>3.0.0</version>
```

Comment:
We're already using this plugin in everywhere else without a version number, why specify it here now?